package.json

```diff
@@ -62,6 +62,7 @@
     "@balena/lint": "^6.1.1",
     "@resin.io/types-mochainon": "^2.0.1",
     "@types/mocha": "^5.2.7",
+    "@types/json-schema": "^7.0.9",
```

Comment:
I think this need to be in the non-dev dependencies, since otherwise users will face issues with compiling TS if they don't already have `@types/json-schema` in their node_modules`.